pytest_factoryboy/fixture.py

```diff
@@ -58,6 +59,19 @@ def __call__(self, request: SubRequest) -> Any:
         return self.function(request)
 
 
+class Box(Generic[T_co]):
+    """Simple box class, used to hold a value.
+
+    The main purpose of this is to hold objects that we don't want to appear in stack traces.
+    For example, the "caller_locals" dict holding a lot of items.
+    """
+
+    value: T_co
```

Comment:
NIP: this is probably not required because we are simply assigning `value: T_co` to `self.value`, so `mypy` and other type checkers will use `T_co` as `self.value` type hinting